components/common/NotificationBox/NotificationBoxModal.vue

```diff
@@ -0,0 +1,285 @@
+          </div>
+        </div>
+      </div>
+      <div v-if="displayedEvents.length === 0" class="empty-tip">
+        <p>{{ $t('notification.emptyTipLine1') }}</p>
+        <p>{{ $t('notification.emptyTipLine2') }}</p>
+      </div>
+      <div v-else class="is-flex is-flex-direction-column">
+        <NotificationItem
+          v-for="(event, index) in displayedEvents"
+          :key="`${event.id}-${index}`"
+          :event="event" />
+      </div>
+    </div>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { Event, FilterOption } from './types'
+import { NeoButton, NeoIcon } from '@kodadot1/brick'
+import NeoTag from '@/components/shared/gallery/NeoTag.vue'
+
+import NotificationItem from './NotificationItem.vue'
+import {
+  Interaction,
+  getInteractionName,
+  useNotification,
+} from './useNotification'
+
+const { $store } = useNuxtApp()
+
+const eventTypes = ref<string[]>([
+  Interaction.SALE,
+  Interaction.OFFER,
+  Interaction.ACCEPTED_OFFER,
+])
+
+const collections = ref<FilterOption[]>([])
+const allEvents = ref<Event[]>([])
+
+const collectionFilter = ref<FilterOption | null>(null)
+const toggleCollectionFilter = (target: FilterOption) => {
+  if (collectionFilter.value?.id === target.id) {
+    collectionFilter.value = null
+  } else {
+    collectionFilter.value = target
+  }
+}
+
+const eventFilter = ref<string[]>([])
+const toggleEventFilter = (target) => {
+  const index = eventFilter.value.findIndex((x) => x === target)
+  if (index === -1) {
+    eventFilter.value.push(target)
+  } else {
+    eventFilter.value.splice(index, 1)
+  }
+}
+
+const showFilter = ref(false)
+const isFilterEmpty = computed(
+  () => !collectionFilter.value && eventFilter.value.length === 0
+)
+
+const displayedEvents = ref<Event[]>([])
+
+const doSearch = () => {
+  displayedEvents.value = allEvents.value.filter(
+    (item) =>
+      (!collectionFilter.value ||
+        collectionFilter.value.id === item.nft.collection?.id) &&
+      (eventFilter.value.length === 0 ||
+        eventFilter.value.some((x) => x === item.interaction))
+  )
+}
+
+const getNotifications = () => {
+  const { collections: collectionData, events: allEventsData } =
+    useNotification($store.getters.getAuthAddress)
+  collections.value = collectionData as unknown as FilterOption[]
+  allEvents.value = allEventsData as unknown as Event[]
+}
+
+watch(allEvents, doSearch, {
+  immediate: true,
+})
+
+watch(collectionFilter, doSearch, { deep: true })
+watch(eventFilter, doSearch, { deep: true })
+
+watch(() => $store.getters.currentUrlPrefix, getNotifications, {
+  immediate: true,
+})
+const emit = defineEmits(['close'])
+</script>
+
+<style scoped lang="scss">
+@import '@/styles/abstracts/variables';
+.no-wrap {
+  white-space: nowrap;
+}
+.rounded {
+  border-radius: 2rem;
+  overflow: hidden;
+}
+.notification-box-modal {
+  .notification-modal-container {
+    position: fixed;
+    top: 0;
+    right: 0;
+    height: 100%;
+    padding-top: 83px;
```

Comment:
please use rem instead. i think 4rem will be suitable here
as a general rule you can round the values from figma to the closest 0.5 rem (8px)

Reply:
hmm i tried 4rem and the ui breaks. it's depends the `$navbar-desktop-min-height` variable it's not a standard value here

Reply:
i see. ok